62_CAD/shaders/main_pipeline/dtm.hlsl

```diff
@@ -0,0 +1,678 @@
+#ifndef _CAD_EXAMPLE_DTM_HLSL_INCLUDED_
+#define _CAD_EXAMPLE_DTM_HLSL_INCLUDED_
+
+#include <nbl/builtin/hlsl/shapes/line.hlsl>
+#include <nbl/builtin/hlsl/algorithm.hlsl>
+
+// TODO: functions outside of the "dtm" namespace need to be moved to another file
+
+// for usage in upper_bound function
+struct StyleAccessor
```

Comment:
move these to line_style.hlsl, and include it here
these aren't related to dtms